geomesa-raster/src/main/scala/org/locationtech/geomesa/raster/data/AccumuloBackedRasterOperations.scala

```diff
@@ -94,13 +102,36 @@ class AccumuloBackedRasterOperations(val connector: Connector,
     writeMutations(createMutation(raster))
   }
 
+  def getMosaicedRaster(query: RasterQuery, params: GeoMesaCoverageQueryParams) = {
+    val rasters = getRasters(query)
+    val (image, numRasters) = profile(RasterUtils.mosaicRasters(rasters,
```

Comment:
Hmm... I think this syntax is perhaps more clear:
```
profile("mosaic") { 
RasterUtils.mosaicRasters(rasters,
  params.height.toInt,
  params.width.toInt,
  params.envelope,
  params.resX,
  params.resY)
}
```
(I believe) This would be enabled by adding this to MethodProfiling
```
def profile[R](identifier: String)(code: => R) = profile(code, identifier)
```
Give that a spin and see if we aren't happier.  It'd add the capability to profile multi-line closures without having to abstract them as functions.